hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -787,6 +787,12 @@
         Allows to configure Memcache text protocol support in Hazelcast.
         -->
         <memcache-protocol enabled="false"/>
+
+        <alto-socket>
```

Comment:
There is a bit of explanation for each tag in this file, I think we should do the same

Reply:
https://github.com/hazelcast/hazelcast/pull/23319/commits/034070382c199cea19e7f577912ba529eaf055f4